Gopkg.toml

```diff
@@ -54,8 +54,8 @@
   version = "1.0.0"
 
 [[constraint]]
-  name = "github.com/satori/go.uuid"
-  version = "1.1.0"
+  name = "github.com/gofrs/uuid"
+  version = "3.2.0"
```

Comment:
In PBS, we use "^3.2.0" to restrict only a major version change.  If a newer minor version change comes along, PBS will take it (e.g., 3.2.1 or 3.3.0 are okay, but 4.0.0 will be disallowed).  Maybe the same should be done here!?

Reply:
Yeah, I agree with @josephveach here

Reply:
Ok. Let me use the ^ modifier